src/main/java/org/broadinstitute/hellbender/tools/spark/pathseq/HostAlignmentReadFilter.java

```diff
@@ -9,17 +9,15 @@
 import java.util.List;
 
 /**
- * Filters out reads above a threshold coverage (number of matches, mismatches, and insertions) and identity
- *   (number of matches less deletions), each given in bases.
+ * Filters out reads above a threshold identity (number of matches less deletions), given in bases.
```

Comment:
The identity is supposed to be the number of bases in the read that match the reference. Even though it doesn't check for soft clips or insertions, they are accounted for because they aren't counted (only matches are). In other words, we could get the same result by taking the read length and subtracting soft clips and insertions. Deletions, on the other hand, do not account for bases in the read, so they need to be handled as a kind of special case. One option would be to ignore them, but it seems more reasonable to penalize them the same as insertions, with -1. 

One may be able to devise weird corner cases where this might be a good scheme, but in practice it works with BWA alignments.